src/states/elder/mod.rs

```diff
@@ -674,14 +676,14 @@ impl Elder {
                 outbox.send_event(content.into_event(src, dst));
                 Ok(())
             }
-            (AckMessage(_sec_info), Section(_src), Section(_dst)) => {
+            (AckMessage(sec_info), Section(src), Section(dst)) => {
                 // TODO: The shared_state shall be updated on when on consensus.
                 //       Which is part of the issue 1670 to handle it.
                 #[cfg(feature = "mock_base")]
                 {
                     self.received_section_info_ack = true;
```

Comment:
We want to remove this now that we receive it, and replace the test with one that covers check that their_knowledge is as expected.